lib/salus/scanners/base.rb

```diff
@@ -461,5 +479,16 @@ def build_options(prefix:, suffix:, separator:, args:, join_by: ',')
         options + option
       end
     end
+
+    def scanner_timeout
+      scanner_timeout_config_param = @config['scanner_timeout_s']
+      # If a developer mistakenly defines this parameter
+      # as a non-integer value, let it be known
+      unless scanner_timeout_config_param.is_a? Integer
```

Comment:
What you have here is fine for now. 
But the `is_a? Integer` will also pass if a negative integer is used, and the code may behave strangely.

Reply:
May change is to something like this?
`x.is_a? Integer && x >= 0`